recipes/lrez/build.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
-export HTSLIB_ROOT=${PREFIX}/lib
-rm -rf build
-mkdir build
-cd build
-cmake ..
-make -j2
-cd ..
 
-mkdir -p $PREFIX/bin/
 
+export C_INCLUDE_PATH=${PREFIX}/include
+export CPLUS_INCLUDE_PATH=${PREFIX}/include
+export LD_LIBRARY_PATH=${PREFIX}/lib
+export LIBRARY_PATH=${PREFIX}/lib
+
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="-lz -lm -lc -lboost_iostreams -L$PREFIX/lib"
+export CPATH=${PREFIX}/include
 
-cp build/tools/Compare ${PREFIX}/bin/
-chmod +x $PREFIX/bin/Compare
+mkdir -p ${PREFIX}/bin
 
-cp build/tools/BamExtractor ${PREFIX}/bin/
-chmod +x $PREFIX/bin/BamExtractor
+# Install bamtools
+cd bamtools
+mkdir -p build
+cd build
+cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$PWD/../ ..
+make
+make install
+cd ../
+if [ -d "lib64" ]
+then
+	mv lib64 lib
+fi
+cp ${SRC_DIR}/bamtools/lib/*.so* ${PREFIX}/lib
```

Comment:
Could you try to add `bamtools` to `requirements/host` instead?

Reply:
I'm using a modified version of bamtools, so I actually need the one which is included in the sources.

Reply:
Could you add a comment to indicate this here?
Also, could you add something like
```yaml
  run_constrained:
    # lzres bundles a custom lib/libbamtools.so which conflicts with the main bamtools package
    - bamtools <0a0
```
to the `requirements` section in `meta.yaml` to prevent file clobbering?

Reply:
And please include BamTools' license information in the `license`/`license_file` fields.

Reply:
Not exactly sure how I should include the license information?

Reply:
See https://github.com/bioconda/bioconda-recipes/pull/28019/files#r613401492